src/main/java/net/masterthought/cucumber/CucumberReportGeneratorMojo.java

```diff
@@ -98,9 +98,21 @@
      */
     private Map<String, String> classifications;
 
+    /**
+     * Set this to "true" to bypass generation of Cucumber Reports entirely.
+     *
+     * @parameter property="skip" default-value="false"
+     */
+    private boolean skip;
```

Comment:
fine if you provide more descriptive field name

Reply:
Makes sense. Changing "skip" to "cucumber.report.skip"